src/commands/block/info.ts

```diff
@@ -46,7 +46,7 @@ export class BlockHeaderTable {
             ['Hash:', blockInfo.hash],
             ['Generation Hash:', blockInfo.generationHash],
             ['Total Fee:', blockInfo.totalFee.toString()],
-            ['Number of Transactions:', blockInfo.numTransactions],
+            ['Number of Transactions:', blockInfo.transactionsCount],
```

Comment:
there is a new property `totalTransactionCount` which includes embedded transactions.  Should cli show both?

Reply:
Added with de92795 :+1: